src/measure_theory/measurable_space.lean

```diff
@@ -350,19 +350,14 @@ iff.refl _
 
 theorem is_measurable_Sup {ms : set (measurable_space α)} {s : set α} :
   @is_measurable _ (Sup ms) s ↔
-    generate_measurable (⋃₀ (measurable_space.is_measurable' '' ms)) s :=
+    generate_measurable {s : set α | ∃ m ∈ ms, @is_measurable _ m s} s :=
```

Comment:
What is the motivation behind this change of statement and the next one?

Reply:
Both changes were made to avoid using `set α → Prop` as `set (set α)`. Also I think that the new statements save extra rewrites of `mem_sUnion`/`mem_Union` both in their proofs, and in lemmas that use them.